persistence/persistence-mybatis/src/main/resources/org/cbioportal/persistence/mybatis/TreatmentMapper.xml

```diff
@@ -33,6 +33,7 @@
             INNER JOIN cancer_study ON patient.CANCER_STUDY_ID = cancer_study.CANCER_STUDY_ID
         <include refid="where"/>
             AND clinical_event_data.KEY = 'SAMPLE_ID'
+            AND (clinical_event.EVENT_TYPE = 'Sample Acquisition' OR clinical_event.EVENT_TYPE = 'SPECIMEN')
```

Comment:
@dippindots maybe good to allow case insensitive here as well as underscores, like: https://github.com/cBioPortal/cbioportal-frontend/pull/4489/files#diff-61b054d9b46310665b83bb6e1754f86f4d019ceadecd86979d51d608080d007aR98-R99